runtime_test.go

```diff
@@ -665,6 +665,49 @@ func TestRuntime_Close_ClosesCompiledModules(t *testing.T) {
 	}
 }
 
+func TestRuntime_Close_Instantiate_ShouldErr(t *testing.T) {
```

Comment:
we don't have this long top leve test name
```suggestion
// TestRuntime_Closed ensure invocation of closed Runtime's methods is safe.
func TestRuntime_Closed(t *testing.T) {
```